lib/node_modules/@stdlib/stats/base/dists/laplace/cdf/benchmark/benchmark.js

```diff
@@ -58,19 +67,24 @@ bench( pkg, function benchmark( b ) {
 bench( pkg+':factory', function benchmark( b ) {
 	var mycdf;
 	var scale;
+	var len;
 	var mu;
 	var x;
 	var y;
 	var i;
 
 	mu = 0.0;
 	scale = 1.5;
+	len = 100;
+	x = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( -2.0, 4 );
```

Comment:
```suggestion
		x[ i ] = uniform( -2.0, 2.0 );
```

This change (to keep the same original range) applies throughout the PR.